chart/templates/metadata-ingress-virtual-service.yaml

```diff
@@ -16,6 +16,6 @@ spec:
   http:
     - route:
       - destination:
-          host: {{ .Release.Name }}-proxy-metadata
+          host: {{ .Release.Name }}-proxy-node-metadata
```

Comment:
This is pointing at the service that is created by the VMC when processing a `Metadata` resource, which might not be what we want. Potentially lengthy discussion to have around this that I'm not going to attempt via github comments.

Reply:
Actually we don't need this anymore - that's the thing I was talking about this morning. Let's discuss tomorrow.